pkg/controller/virtinformers.go

```diff
@@ -450,3 +453,16 @@ func (f *kubeInformerFactory) OperatorInstallStrategyJob() cache.SharedIndexInfo
 		return cache.NewSharedIndexInformer(lw, &batchv1.Job{}, f.defaultResync, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
 	})
 }
+
+func (f *kubeInformerFactory) OperatorPod() cache.SharedIndexInformer {
+	return f.getInformer("operatorPodsInformer", func() cache.SharedIndexInformer {
+		// Watch all kubevirt infrastructure pods with the operator label
+		labelSelector, err := labels.Parse(OperatorLabel)
+		if err != nil {
+			panic(err)
+		}
+
+		lw := NewListWatchFromClient(f.clientSet.CoreV1().RESTClient(), "pods", k8sv1.NamespaceAll, fields.Everything(), labelSelector)
```

Comment:
Do you use all namespaces in order to detect parallel installations and allow namespace changes?

Reply:
kubevirt can be installed in any namespace.